### ISSUE TITLE
[Bug]: No Light Levels [1.19]

### ISSUE THREAD

**Issue by NFLD99** [cominixo/BetterF3#95]
### Describe the Bug

Light levels dont show on 1.19

### Steps to Reproduce

install better f3, 
press f3

### Expected Behavior

Light levels show in f3

### Screenshots

With Better f3: ![With Better f3](https://uploads.nfld99.com/wbf3.png)

Without Better f3: ![Without Better f3](https://uploads.nfld99.com/wobf3.png)

### Modloader Type

Fabric

### Modloader Version

fabric-loader-0.14.8-1.19

### Mod Version

BetterF3-1.3.1-Fabric-1.19

### Log Output

```shell
N/A
```

### Additional Context

Mod List: 
```
    Adorn-3.5.4+1.19-fabric.jar
    animatica-0.5+1.19.jar
    architectury-5.7.28-fabric.jar
    badpackets-fabric-0.1.2.jar
    BetterF3-1.3.1-Fabric-1.19.jar
    bettermounthud-1.1.4.jar
    Bookshelf-Fabric-1.19-14.0.3.jar
    borderless-mining-1.1.4+1.19.jar
    carpet-extra-1.19-1.4.79.jar
    cloth-config-7.0.72-fabric.jar
    continuity-2.0.0+1.19.jar
    Cosmetica 1.19 1.1.1.jar
    cullleaves-2.3.4.jar
    easiervillagertrading-1.19-fabric0.53.3-1.5.4.jar
    EnchantmentDescriptions-Fabric-1.19-11.0.4.jar
    enhancedblockentities-0.7.1+1.19.jar
    entity-texture-features-3.1.4-mc1.19.jar
    fabric-api-0.56.3+1.19.jar
    fabric-carpet-1.19-1.4.79+v220607.jar
    fabric-language-kotlin-1.8.1+kotlin.1.7.0.jar
    Female-Gender-Mod-fabric-1.19-3.0.0.jar
    fpsdisplay-1.5.0+1.19.jar
    horse-stats-vanilla-4.2.0.jar
    immersive-portals-2.0.4-mc1.19-fabric.jar
    indium-1.0.6+mc1.19.jar
    infinitevillagertrading-1.19-fabric0.53.3-1.1.jar
    iris-mc1.19.1-rc1-1.2.6-rc.1.jar
    kubejs-fabric-1900.5.5-build.23.jar
    kubejs-ui-fabric-1900.1.8-build.20.jar
    minecraft-comes-alive-7.3.4+1.19-fabric.jar
    modmenu-4.0.0.jar
    MouseTweaks-fabric-mc1.19-2.22.jar
    PerfectAccuracy-1.2-1.19.jar
    plushie-mod-1.2.jar
    rhino-fabric-1802.1.14-build.190.jar
    RoughlyEnoughItems-9.1.500.jar
    sodium-fabric-mc1.19-0.4.2+build.16.jar
    starlight-1.1.1+fabric.ae22326.jar
    statuseffecttimer-1.1.1+1.19.jar
    TaxFreeLevels-1.3.jar
    wthit-fabric-5.5.1.jar
    XaerosWorldMap_1.23.3_Fabric_1.19.jar
    Xaeros_Minimap_22.9.3_Fabric_1.19.jar
    Zoomify-1.7.1.jar
```

**Comment by TreyRuffy** [comment]
I believe the issue is with immersive portals and I have yet to find a fix for this.

**Comment by TreyRuffy** [comment]
Fixed in 4.0.0
